addons/l10n_pt_stock/models/stock_warehouse.py

```diff
@@ -0,0 +1,24 @@
+from odoo import models, api, _
+
+
+class Warehouse(models.Model):
+    _inherit = "stock.warehouse"
+
+    def _get_sequence_values(self, name=False, code=False):
+        sequence_values = super()._get_sequence_values(name=name, code=code)
+        if self.company_id.country_id.code != 'PT':
+            return sequence_values
+        sequence_values.get("out_type_id").update({
+            'prefix': _("TMP/"),
```

Comment:
Prefixes and suffixes are not meant to be translatable as it can result of different picking names.
Also if you have 2 warehouses having their own delivery picking type, they will have the same sequence